### PR TITLE
Copter: Simulink Model and init scripts for Copter-4.3

### PR DESCRIPTION
**Pull request by fbredeme**
- arducopter.slx: Simulates ArduCopter Stabilize and Althold controller and optional plant model
- sid_pre.m: Loads *.bin files to Matlab structs
- sid_sim_init.m: Loads signals and parameters from Matlab structure into Simulink model
- sid_controller_validation.m: Validation of the flight controller model with the flight data loaded to the Matlab workspace.

### CODE REVIEW

**Review by amilcarlucas — APPROVED** [review]

**Comment by amilcarlucas** [comment]
Once this gets merged you should do the same pull request, but against master

File: Tools/simulink/arducopter/functions/getMotorMixerFactors.m
Comment (rmackay9) [inline Tools/simulink/arducopter/functions/getMotorMixerFactors.m:12]: no big deal but I wonder if these ^M characters could be removed?

**Comment by rmackay9** [comment]
Thanks for that.  I'll add it to the dev call but I think we don't do much code review so I suspect it will go in as-is.

**Review by tridge — APPROVED** [review]

**Comment by tridge** [comment]
@fbredeme thanks! will merge when CI passes

**Pull request merged**
